### PR TITLE
Savva Dzhumaliev Tasks 11-20

### PR DESCRIPTION
**Pull request by EasileleM**

### CODE REVIEW

File: tasks/12_title-case/title-case.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function titleCase(str) {
+  let arr = str
```
Comment (akimovaa) [inline tasks/12_title-case/title-case.js]: const instead let

File: tasks/12_title-case/title-case.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+      .toLowerCase()
+      .split(' ');
+  const changeWord = (w, i) => arr[i] = w.charAt(0).toUpperCase() + w.slice(1);
```
Comment (akimovaa) [inline tasks/12_title-case/title-case.js]: what does <code>w, i, arr</code> mean? Make naming of variables cleaner.

File: tasks/13_largest-of-four/largest-of-four.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export function largestOfFour(arr) {
+  const result = [];
+  const findMax = (max, num) => Math.max(max, num);
+  arr.forEach(function(subArr) {
```
Comment (akimovaa) [inline tasks/13_largest-of-four/largest-of-four.js:3]: Try to use es6

File: tasks/13_largest-of-four/largest-of-four.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    if (subArr.length === 0) {
+      result.push(NaN);
+    } else {
+      result
+          .push(subArr.reduce(findMax, -Infinity));
```
Comment (akimovaa) [inline tasks/13_largest-of-four/largest-of-four.js]: useless <code>-Infinity</code>. Read spec of <code>reduce()</code>.

File: tasks/13_largest-of-four/largest-of-four.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+export function largestOfFour(arr) {
+  const result = [];
+  const findMax = (max, num) => Math.max(max, num);
+  arr.forEach(function(subArr) {
+    if (subArr.length === 0) {
```
Comment (akimovaa) [inline tasks/13_largest-of-four/largest-of-four.js]: <code>!subArr.length</code>instead <code>subArr.length === 0</code>

File: tasks/14_clone-arr/clone-arr.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+export function cloneArr(arr) {
+  return arr.slice();
```
Comment (akimovaa) [inline tasks/14_clone-arr/clone-arr.js]: It's a shallow copy. 

<code>var b = [-4,-3, {d:1}];
var a = cloneArr(b);
console.log(a[2]===b[2]); //true
</code>

File: tasks/19_array-chunks/array-chunks.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+export function arrayChunks(arr, size) {
```
Comment (akimovaa) [inline tasks/19_array-chunks/array-chunks.js:1]: Try to use reducer at this case

File: tasks/20_truncate-array/truncate-array.js [hunk 8]
```diff
@@ -0,0 +1,6 @@
+// Should i use pure function in this case?
+export function truncateArray(arr, howMany) {
```
Comment (akimovaa) [inline tasks/20_truncate-array/truncate-array.js:1]: This function is a pure, you don't need to modify <code>arr</code>.

**Review by alexkond — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Review by akimovaa — APPROVED** [review]

**Pull request merged**